openshift2nulecule/openshift.py

```diff
@@ -162,17 +163,20 @@ def __init__(self, artifacts):
 
         # get all images of all ReplicationControllers
         self.images = []
-        for artifact in self.artifacts:
-            # TODO: add support for other kinds (Pod, ....?)
-            if artifact["kind"] in ["ReplicationController", "DeploymentConfig"]:
-                self.images.extend(utils.get_image_info(artifact))
+        for provider in NULECULE_PROVIDERS:
+            for artifact in self.artifacts[provider]:
+                # TODO: add support for other kinds (Pod, ....?)
+                if artifact["kind"] in ["ReplicationController",
+                                        "DeploymentConfig"]:
+                    self.images.extend(utils.get_image_info(artifact))
 
     def _remove_securityContext(self):
         """
-        Remove securityContext from all objects in kind_list.
+        Remove securityContext from all objects in kind_list
+        for Kubernetes artifacts.
         """
 
-        for obj in self.artifacts:
+        for obj in self.artifacts["kubernetes"]:
```

Comment:
I am skeptical about this due to two reasons.
1. When I was testing before I also thought of running this only for k8s, but then it failed in some places (might be this was not the reason)

2. We are not exporting the pods, so do we even need the whole function? Need to do some extensive testing.

Correct me if I am wrong!